### PR TITLE
add and modify translation

### PR DESCRIPTION
**Pull request by vickyhella3**

### CODE REVIEW

File: assets/translations/zh-hans.yaml [hunk 1]
```diff
@@ -1780,17 +2495,17 @@ members:
     viewNodes: 查看节点
     owner:
       label: 所有者
-      description: 所有者对集群和集群内的所有资源拥有完全的控制权.
+      description: 所有者对集群和集群内的所有资源拥有完全的控制权。
     member:
       label: 成员
-      description: 成员可以管理集群内的资源, 但不能改变群集本身.
+      description: 成员可以管理集群内的资源，但不能改变群集本身。
```
Comment (n313893254) [inline assets/translations/zh-hans.yaml]: `群集` should be `集群`

File: assets/translations/zh-hans.yaml [hunk 2]
```diff
@@ -1800,25 +2515,25 @@ monitoring:
...
     enable:
       label: 部署 Alertmanager
     secrets:
       additional:
-        info: "密文应挂载到容器路径 <pre class='inline-block m-0'>/etc/alertmanager/secrets/</pre>。"
-        label: 附加密文
-      existing: 选择现有的配置密文
+        info: Secret 应挂载到容器路径 <pre class='inline-block m-0'>/etc/alertmanager/secrets/</pre>
+        label: 补充 Secret
+      existing: 选择现有的配置 Secret
```
Comment (n313893254) [inline assets/translations/zh-hans.yaml]: `Secret` can be translate into `密文`

**Review by n313893254 — CHANGES_REQUESTED** [review]

File: assets/translations/zh-hans.yaml [hunk 3]
```diff
@@ -1223,34 +1793,33 @@ fleet:
...
 footer:
   docs: Rancher 官方文档
   download: 下载 CLI
   forums: 论坛
   issue: 提交 GitHub Issue
   slack: Slack 讨论群
-  wechat: 
+  wechat:
     title: 微信
-    modalText: 扫码关注 Rancher 中国微信技术群
```
Comment (kingsd041) [inline assets/translations/zh-hans.yaml]: Missing `modalText: 扫码关注 Rancher 中国微信技术群`

File: assets/translations/zh-hans.yaml [hunk 4]
```diff
@@ -1223,34 +1793,33 @@ fleet:
...
         }
 footer:
   docs: Rancher 官方文档
   download: 下载 CLI
   forums: 论坛
   issue: 提交 GitHub Issue
   slack: Slack 讨论群
-  wechat: 
+  wechat:
     title: 微信
```
Comment (kingsd041) [inline assets/translations/zh-hans.yaml]: Missing `modalText: 扫码关注 Rancher 中国微信技术群`

Comment (kingsd041) [inline assets/translations/zh-hans.yaml]: Missing `modalText: 扫码关注 Rancher 中国微信技术群`

File: assets/translations/zh-hans.yaml [hunk 5]
```diff
@@ -1436,112 +2007,227 @@ istio:
...
+    docs: Rancher 官方文档
     forums: 论坛
   commercial:
-    title: 付费支持
+    title: 商业支持
     body: 了解商业支持
   landingPrefs:
-    title: 你想在登录时看到什么？
-    body: "你可以改变你登陆时的位置:"
+    title: 选择登录后显示页面
```
Comment (kingsd041) [inline assets/translations/zh-hans.yaml]: It should be `选择登录后显示的页面`

File: assets/translations/zh-hans.yaml [hunk 6]
```diff
@@ -2415,20 +3171,22 @@ persistentVolumeClaim:
...
 prefs:
-  title: 用户偏好设置
+  title: 偏好设置
   theme:
     label: 主题
     light: 浅色
     auto: 自动
     dark: 深色
-    autoDetail: 选择自动设置，将会在晚 6 点到次日早 6 点间自动切换到黑色主题。
+    autoDetail: 自动模式使用系统设置，或在晚{pm}点到次日早{am}点间自动切换到深色模式。
```
Comment (kingsd041) [inline assets/translations/zh-hans.yaml]: Need to be replaced by `自动模式使用系统设置，或在{pm}到次日{am}间自动切换到深色模式。`

File: assets/translations/zh-hans.yaml [hunk 7]
```diff
@@ -2438,7 +3196,7 @@ prefs:
     value: |-
       {count, number}
   keymap:
-    label: YAML 编辑器选择
+    label: YAML 编辑器键映射
```
Comment (kingsd041) [inline assets/translations/zh-hans.yaml]: should be `YAML 编辑器`

File: assets/translations/zh-hans.yaml [hunk 8]
```diff
@@ -4916,170 +5976,295 @@ unit:
...
-    learnMore: 了解更多关于SUSE Rancher支持的信息
+    learnMore: 进一步了解 SUSE Rancher 支持。
+    pricing: 联系我们获取价格
+  subscription:
+    haveSupport: 已获得支持？
+    addSubscription: 添加订阅 ID
+    removeSubscription: 移除订阅 ID
+    addTitle: 添加 SUSE 订阅 ID
+    addLabel: "请输入有效的订阅 ID："
+    removeTitle: 移除 ID？
```
Comment (yzeng25) [inline assets/translations/zh-hans.yaml]: `是否移除订阅 ID？`

File: assets/translations/zh-hans.yaml [hunk 9]
```diff
@@ -4916,170 +5976,295 @@ unit:
...
+  replicationControllers: ReplicationController
+  requestsCpu: CPU 预留
   requestsMemory: 内存预留
   requestsStorage: 存储预留
   secrets: 密文
   services: 服务
   servicesLoadBalancers: 服务负载均衡
   servicesNodePorts: 服务节点端口
+  namespaceLimit:
+    label: 限制
```
Comment (yzeng25) [inline assets/translations/zh-hans.yaml]:
```suggestion
    label: 命名空间限制
```

File: assets/translations/zh-hans.yaml [hunk 10]
```diff
@@ -4916,170 +5976,295 @@ unit:
...
+    consent: 同意横幅
+  color:
+    label: 主颜色
+    tip: 使用自定义颜色替换整个 UI 中使用的主颜色。
+    useCustom: 使用自定义颜色
+  linkColor:
+    label: 链接颜色
+    tip: 使用自定义链接颜色替换整个 UI 中使用的链接颜色。
+    useCustom: 使用自定义链接颜色
+    example: 链接示例
```
Comment (yzeng25) [inline assets/translations/zh-hans.yaml]: seems like this empty line can be removed

File: assets/translations/zh-hans.yaml [hunk 11]
```diff
@@ -4916,170 +5976,295 @@ unit:
...
+    tip: '上传 Logo 以替换顶层导航标题中的 Rancher logo。图片的高度应为 21 像素，最大宽度为 200 像素。最大文件大小为 20KB'
+    lightPreview: 浅色主题预览
+    darkPreview: 深色主题预览
+    uploadLight: 上传浅色 Logo
+    uploadDark: 上传深色 Logo
+    useCustom: 使用自定义 Logo
+  options:
+    default: 默认 Rancher 主题
+    suse: SUSE 主题
+    custom: 定义自定义主题
```
Comment (yzeng25) [inline assets/translations/zh-hans.yaml]:
```suggestion
    custom: 自定义主题
```

File: assets/translations/zh-hans.yaml [hunk 12]
```diff
@@ -4916,170 +5976,295 @@ unit:
...
+    'auth-user-session-ttl-minutes': '用户认证会话的自定义 TTL（单位：分钟）。'
+    'auth-token-max-ttl-minutes': '身份验证 Token 的自定义最大 TTL（单位：分钟）。'
+    'kubeconfig-generate-token': '自动为用户生成 Kubeconfig Token。'
+    'kubeconfig-token-ttl-minutes': 'Kubeconfig Token 的自定义最大 TTL（单位：分钟）。'
+    'rke-metadata-config': '配置 RKE 元数据刷新参数。'
+    'ui-banners': '分类横幅用于在页眉、页脚或两者中显示自定义的固定横幅。'
+    'ui-consent-banner': '横幅用于在登录期间，向用户展示自定义同意横幅。'
+    'ui-default-landing': '用户登录后的默认页面。'
+    'brand': 在'/assets/brand'中定义，用于替代主题的文件夹名称
+    'harv-vlan': VLAN 网络的默认网络接口名称。
```
Comment (yzeng25) [inline assets/translations/zh-hans.yaml]:
```suggestion
    'harv-vlan': 'VLAN 网络的默认网络接口名称。'
```

Comment (yzeng25) [reply assets/translations/zh-hans.yaml]: Double check lines from 6019 to 6056. The formats are somewhat confusing here.
Which one is correct? Or maybe all of them are correct? I would suggest keeping them in one form.
`'a':'descrption'`
`'a':descrption`
`'a':"descrption"`

File: assets/translations/zh-hans.yaml [hunk 13]
```diff
@@ -4916,170 +5976,295 @@ unit:
...
   edit:
     label: 编辑设置
-    changeSetting: "修改设置"
+    changeSetting: "修改设置："
     trueOption: "True"
     falseOption: "False"
     value: 值
     useDefault: 使用默认值
-    invalidJSON: 无效的JSON - 请在保存前检查并修改您的输入。
+    invalidJSON: 无效的 JSON - 请先检查并修改你的输入，然后再保存。
```
Comment (yzeng25) [inline assets/translations/zh-hans.yaml]:
```suggestion
    invalidJSON: 无效的 JSON - 请先检查并修改你输入的值，然后再保存。
```

File: assets/translations/zh-hans.yaml [hunk 14]
```diff
@@ -4741,48 +5729,59 @@ typeLabel:
...
     {count, plural,
       =1 { 集群 }
-      other { 集群 }
+      other {集群 }
     }
   fleet.cattle.io.clustergroup: |-
     {count, plural,
       one { 集群组 }
-      other { 集群组 }
+      other {集群组 }
```
Comment (yzeng25) [inline assets/translations/zh-hans.yaml]:
```suggestion
      other { 集群组 }
```

File: assets/translations/zh-hans.yaml [hunk 15]
```diff
@@ -4741,48 +5729,59 @@ typeLabel:
...
-      one { Endpoint }
-      other { Endpoints }
+      one { 端点 }
+      other { 端点 }
     }
   fleet.cattle.io.cluster: |-
     {count, plural,
       =1 { 集群 }
-      other { 集群 }
+      other {集群 }
```
Comment (yzeng25) [inline assets/translations/zh-hans.yaml]:
```suggestion
      other { 集群 }
```

**Review by n313893254 — APPROVED** [review]
LGTM

File: assets/translations/zh-hans.yaml (hunk 12 above)
Comment (vickyhella3) [reply assets/translations/zh-hans.yaml]: The current ones follow the formats of the original English version, so I think it should be fine~

File: assets/translations/zh-hans.yaml (hunk 10 above)
Comment (vickyhella3) [reply assets/translations/zh-hans.yaml:6153]: The English version also has this line~

File: assets/translations/zh-hans.yaml (hunk 12 above)
Comment (yzeng25) [reply assets/translations/zh-hans.yaml:6047]: cool!

File: assets/translations/zh-hans.yaml (hunk 10 above)
Comment (yzeng25) [reply assets/translations/zh-hans.yaml:6153]: Sounds good. Keep it then. Other LGTM :)

**Comment by richard-cox** [comment]
@n313893254 / @kingsd041 It looks like the last set of changes came in following your reviews, is this PR still ok to merge?

**Pull request merged**